### PR TITLE
fix(HMS-951): region refresh docs

### PR DESCRIPTION
**Pull request by lzap**
This is a follow-up PR for https://github.com/RHEnVision/provisioning-backend/pull/433

It adds a documentation of the process, also added a missing newline from the std output.

### CODE REVIEW

File: docs/preloading-cloud-data.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+AZURE_CLIENT_ID=UUID
+AZURE_CLIENT_SECRET=SECRET
+GCP_JSON=xxxxxxx
+GCP_PROJECT_ID=myproject-12345
+```
+
+Important: AWS requires special attention: global endpoint session setting must be set for an account. 
+
+1. Sign in as a root user or a user with permissions to perform IAM administration tasks. To change the compatibility of session tokens, you must have a policy that allows the iam:SetSecurityTokenServicePreferences action.
+2. Open the IAM console. In the navigation pane, choose Account settings.
```
Comment (adiabramovitch) [inline docs/preloading-cloud-data.md:56]: typo

File: docs/preloading-cloud-data.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+The terminology is different for each hyperscaler, however, the data model uses the AWS terminology: regions and (availability) zones:
+
+* AWS EC2: zones (`us-east-1`) and availability zones (`a`), fully qualified name is `us-east-1a`
+* Azure: locations (`northeurope`) and availability zones (`1`), fully qualified name is `northeurope_1`
+* GCP: zones (`us-west`) and regions (`b`), fully qualified name is `us-west-b`
+
+### Registered Instance Types
+
+The `InstanceType` type contains common fields like number of vCPUs and cores, total amount of memory, ephemeral storage and supported flag. It also has an architecture, therefore, a single instance type cannot have multiple architectures. In this model, every instance type with a different architecture has a different instance type name. This applies to all major hyperscalers except Azure where Intel 64bit and 32bit are shared. However, 32bit is no longer supported by RHEL and therefore this is not an issue.
```
Comment (adiabramovitch) [inline docs/preloading-cloud-data.md:23]:
```suggestion
The `InstanceType` type contains common fields like the number of vCPUs and cores, total amount of memory, ephemeral storage, and supported flag. It also has an architecture, therefore, a single instance type cannot have multiple architectures. In this model, every instance type with a different architecture has a different instance type name. This applies to all major hyperscalers except Azure where Intel 64bit and 32bit are shared. However, 32bit is no longer supported by RHEL and therefore this is not an issue.
```

File: docs/preloading-cloud-data.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+If these steps are not done, the operation will throw AuthFailure "AWS was not able to validate the provided access credentials". Our shared (team) account, unfortunately, will not work. The solution is to use personal account, create a `typesctl.env` config file which will override `api.env`:
+
+```
+# cat config/typesctl.env
+AWS_KEY=PERSONAL_ACCOUNT_KEY
+AWS_SECRET=PERSONAL_ACCOUNT_SECRET
+```
+
+Make sure you either use root account, or the role has necessary permissions (`DescribeRegions` and others). You can find these permissions in the [AWS configuration document](./configure-amazon-role.md).
```
Comment (adiabramovitch) [inline docs/preloading-cloud-data.md:82]:
```suggestion
Make sure you either use the root account, or the role has the necessary permissions (`DescribeRegions` and others). You can find these permissions in the [AWS configuration document](./configure-amazon-role.md).
```

**Review by adiabramovitch — APPROVED** [review]
Just some nits 
looks great, thank you!

File: docs/preloading-cloud-data.md (hunk 1 above)
Comment (lzap) [reply docs/preloading-cloud-data.md:56]: Where? (I copy pasted this from AWS docs.)

**Comment by lzap** [comment]
Fixed all thanks, except the "typo" not sure what is wrong.

File: docs/preloading-cloud-data.md (hunk 1 above)
Comment (adiabramovitch) [reply docs/preloading-cloud-data.md:56]: pane ?

**Pull request merged**